<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DirWidget</class>
 <widget class="QWidget" name="DirWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0" columnstretch="0,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>1</number>
   </property>
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <item row="0" column="0" rowspan="2" colspan="2">
    <layout class="QHBoxLayout" name="toolbar_layout"/>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="tool_new_file">
         <property name="statusTip">
          <string>Create a new file</string>
         </property>
         <property name="text">
          <string>New File</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tool_new_dir">
         <property name="statusTip">
          <string>Create a new directory</string>
         </property>
         <property name="text">
          <string>New Dir</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tool_goToImages">
         <property name="statusTip">
          <string>Add selected images to slideshow</string>
         </property>
         <property name="text">
          <string>Slideshow</string>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tool_goToPlayer">
         <property name="statusTip">
          <string>Enqueue selection in multimedia player</string>
         </property>
         <property name="text">
          <string>Play</string>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_status">
       <property name="text">
        <string notr="true">Status</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <layout class="QVBoxLayout" name="browser_layout">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="QFrame" name="group_snaps">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item>
         <widget class="QLabel" name="label_snap">
          <property name="text">
           <string notr="true">Snap</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="slider_snap">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="invertedControls">
           <bool>false</bool>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tool_snap_older">
          <property name="text">
           <string notr="true">...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tool_snap_newer">
          <property name="text">
           <string notr="true">...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="group_actions">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="tool_act_run">
        <property name="toolTip">
         <string>Open item</string>
        </property>
        <property name="statusTip">
         <string>Open item</string>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tool_act_runwith">
        <property name="toolTip">
         <string>Open item (select application)</string>
        </property>
        <property name="statusTip">
         <string>Open item (select application)</string>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tool_act_fav">
        <property name="toolTip">
         <string>Add item to personal favorites</string>
        </property>
        <property name="statusTip">
         <string>Add item to personal favorites</string>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tool_act_rename">
        <property name="toolTip">
         <string>Rename item</string>
        </property>
        <property name="statusTip">
         <string>Rename item</string>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tool_act_cut">
        <property name="toolTip">
         <string>Cut items</string>
        </property>
        <property name="statusTip">
         <string>Cut items (add to the clipboard)</string>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tool_act_copy">
        <property name="toolTip">
         <string>Copy items</string>
        </property>
        <property name="statusTip">
         <string>Copy items to the clipboard</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tool_act_paste">
        <property name="toolTip">
         <string>Paste items from clipboard</string>
        </property>
        <property name="statusTip">
         <string>Paste items from clipboard</string>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tool_act_rm">
        <property name="toolTip">
         <string>Delete Items</string>
        </property>
        <property name="statusTip">
         <string>Delete Items</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionBack">
   <property name="text">
    <string notr="true">Back</string>
   </property>
   <property name="iconText">
    <string>Back</string>
   </property>
   <property name="toolTip">
    <string>Go back to previous directory</string>
   </property>
   <property name="statusTip">
    <string>Go back to previous directory</string>
   </property>
  </action>
  <action name="actionUp">
   <property name="text">
    <string notr="true">Up</string>
   </property>
   <property name="iconText">
    <string>Up</string>
   </property>
   <property name="toolTip">
    <string>Go to parent directory</string>
   </property>
   <property name="statusTip">
    <string>Go to parent directory</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="text">
    <string notr="true">Home</string>
   </property>
   <property name="iconText">
    <string>Home</string>
   </property>
   <property name="toolTip">
    <string>Go to home directory</string>
   </property>
   <property name="statusTip">
    <string>Go to home directory</string>
   </property>
  </action>
  <action name="actionStopLoad">
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="statusTip">
    <string>Stopl loading the directory</string>
   </property>
  </action>
  <action name="actionClose_Browser">
   <property name="text">
    <string notr="true">Close Browser</string>
   </property>
   <property name="iconText">
    <string notr="true">Close Browser</string>
   </property>
   <property name="toolTip">
    <string>Close this browser</string>
   </property>
   <property name="statusTip">
    <string>Close this browser</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
